emukit/bayesian_optimization/loops/bayesian_optimization_loop.py

```diff
@@ -48,3 +54,12 @@ def __init__(self, model: IModel, space: ParameterSpace, acquisition: Acquisitio
         loop_state = create_loop_state(model.X, model.Y)
 
         super().__init__(candidate_point_calculator, model_updaters, loop_state)
+
+    def get_current_best(self) -> dict:
+        """
+        Returns a dictionary with the current best found location (minimum) and its value.
```

Comment:
Could you put the key names in the docstring?

Reply:
Done